flood_adapt/integrator/sfincs_adapter.py

```diff
@@ -225,17 +225,18 @@ def add_dis_bc(self, df_ts: pd.DataFrame):
         """
 
         # Determine bnd points from reference overland model
-        gdf_locs = self.sf_model.forcing["dis"].vector.to_gdf()
-        gdf_locs.crs = self.sf_model.crs
+        # ASSUMPTION: Order of the rivers is the same as the site.toml file
+        if list_df is not None:
```

Comment:
if list_df is enough

Reply:
That's not enough but the following works:
if np.any(list_df):